ideascube/monitoring/forms.py

```diff
@@ -149,3 +151,37 @@ def clean_loan(self):
             # Should araise only after migration 0.3.2 to 0.3.0.
             loan = Loan.objects.due().filter(specimen__barcode=barcode).first()
         return loan
+
+
+class StockImportForm(forms.Form):
+    source = forms.FileField(label=_('CSV File'), required=True)
+
+    def save(self):
+        source = self.cleaned_data['source']
+        source = TextIOWrapper(source.file, encoding='utf-8')
```

Comment:
Watch out that we can still have non UTF-8 CSV. `to_unicode` helpers should be here to handle the most common cases (latin1…).